transforms/unchain-variables.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Unchains chained variable declarations.
+ */
+module.exports = function(file, api) {
+  const jscodeshift = api.jscodeshift;
+
+  const chainedDeclarations = jscodeshift(file.source)
+    .find(jscodeshift.VariableDeclaration)
+    .filter(variableDeclaration => (
+      variableDeclaration.value.declarations.length > 1
+    ));
+
+  chainedDeclarations.forEach(chainedDeclaration => {
+    const kind = chainedDeclaration.value.kind; // e.g. const, let, or var
+
+    jscodeshift(chainedDeclaration)
+      .replaceWith(chainedDeclaration.value.declarations.map((declaration, i) => {
+        const unchainedDeclaration =
+          jscodeshift.variableDeclaration(kind, [declaration]);
+
+        if (i === 0) {
+          unchainedDeclaration.comments = chainedDeclaration.value.comments;
+        } else if (declaration.comments) {
+          unchainedDeclaration.comments = declaration.comments;
+          declaration.comments = null;
+        }
+
+        return unchainedDeclaration;
+      }));
+  });
+
+  return chainedDeclarations.size
+    ? chainedDeclarations.toSource({quote: 'single'})
```

Comment:
Would be good to pass thought print options, as per https://github.com/cpojer/js-codemod/blob/master/transforms/object-shorthand.js#L66